superbench/runner/ansible.py

```diff
@@ -59,27 +60,35 @@ def __init__(self, config):
                 self._config['cmdline'] += ' --ask-pass --ask-become-pass'
         logger.info(self._config)
 
-    def run(self, ansible_config, sudo=False):    # pragma: no cover
+    def run(self, ansible_config, sudo=False, stdout=False):    # pragma: no cover
         """Run Ansible runner.
 
         Args:
             ansible_config (dict): Ansible config dict.
             sudo (bool): Run as sudo or not. Defaults to False.
+            stdout (bool): return stdout or not. Defaults to False.
 
         Returns:
             int: Ansible return code.
+            outputs: Ansible stdout if stdout is True.
         """
         if sudo:
             logger.info('Run as sudo ...')
             ansible_config['cmdline'] += ' --become'
         with tempfile.TemporaryDirectory(prefix='ansible') as tmpdir:
             r = ansible_runner.run(private_data_dir=tmpdir, **ansible_config)
             logger.debug(r.stats)
+            if stdout:
+                raw_outputs = r.stdout.read()
         if r.rc == 0:
             logger.info('Run succeed, return code {}.'.format(r.rc))
         else:
             self.failure_count += 1
             logger.warning('Run failed, return code {}.'.format(r.rc))
+        if stdout:
+            regex = re.compile(r'\x1b(\[.*?[@-~]|\].*?(\x07|\x1b\\))')
+            outputs = regex.sub('', raw_outputs)
+            return outputs
```

Comment:
change the rc code to others may introduce bug in local mode [runner.py L445](https://github.com/microsoft/superbenchmark/blob/main/superbench/runner/runner.py#L445)

Reply:
Thanks for the comment. 
Actually, this feature is just for adding a use for others, people can set stdout as True to get output from ansible runner.
If I understand correctly,  I don't think it could introduce an issue for existing code, as stdout's value is always False.

Reply:
ok, just my concerns, I'm not sure where this feature will be used for, pls at least add test for this feature